interface/README.md

```diff
@@ -0,0 +1,51 @@
+# LXD Interface
+
+Juju interface for LXD built using the [Operators Framework](https://github.com/canonical/operator).
+Enables relations between a LXD cluster or LXD nodes and clients.
+
+### Provider side
+The provider side of the interface writes information about the available LXD nodes on the relation, and
+registers client certificates that it received on the relation to LXD.
+
+### Require side
+The require side of the interface sends client certificates to add to LXD and gets information about
+available LXD nodes on the relation.
+
+## API
+Below is the API that should be used by charms using the LXD interface.
+Each section details the information each role should **publish**.
+The opposite role would then of course have to read data on the relation coming
+from the other side and work with it.
+
+Data for each key should be published as JSON.
+
+#### Provider
+```yaml
+"nodes": [
+    "https://10.10.10.10:8443",
+    "https://10.10.10.11:8444"
+],
+"registered_certificates": [
```

Comment:
Why do we provide a list of registered certificates?

Reply:
It's used to notify members of the relation so they may know when the certificate has been trusted